Add ItchySats App

1010Tom opened the pull request:
ItchySats enables trustless derivatives on Bitcoin using DLCs.

1010Tom committed:
Add ItchySats App

da-kami commented on apps/registry.json:
```diff
@@ -615,5 +615,30 @@
+        "support": "https://github.com/itchysats/itchysats/issues",
+        "port": 7331,
+        "gallery": [
+            "1.jpg",
```
I think those should either be provided of removed

da-kami commented on apps/itchysats/docker-compose.yml:
```diff
@@ -0,0 +1,24 @@
+services:
+  web:
+    # TODO: get latest release from https://github.com/itchysats/itchysats/pkgs/container/itchysats%2Ftaker
+    image: docker pull ghcr.io/itchysats/itchysats/taker:latest@sha256:dba750409d9446ba34ac5682a9bb7f44b9084f506d1971203773e01ba282c361
```
```suggestion
    image: ghcr.io/itchysats/itchysats/taker:latest@sha256:dba750409d9446ba34ac5682a9bb7f44b9084f506d1971203773e01ba282c361
```

1010Tom commented on apps/itchysats/docker-compose.yml:
```diff
@@ -0,0 +1,24 @@
+      - ${APP_DATA_DIR}/data:/data
+    command:
+      - --maker=mainnet.itchysats.network:9999
+      - --maker-id=69a42aa90da8b065b9532b62bff940a3ba07dbbb11d4482c7db83a7e049a9f1e
```
The correct id is: 
```suggestion
      - --maker-id=7e35e34801e766a6a29ecb9e22810ea4e3476c2b37bf75882edf94a68b1d9607
```

da-kami commented:
Install the plugin when being in `umbrel` folder on the raspi:

```
sudo scripts/update/update --repo itchysats/umbrel#itchysats
```

itchymax commented:
How to open PR: https://github.com/getumbrel/umbrel/blob/master/apps/README.md

lukechilds committed:
Autorecover from Docker failure during update (#1154)

lukechilds committed:
Improved Docker error handling (#1155)

lukechilds committed:
Correctly show update errors in web ui (#1157)

lukechilds committed:
Update specter-desktop app to v1.7.2 (#1158)

lukechilds committed:
Update btc-rpc-explorer app to 3.3.0 (#1159)

lukechilds committed:
Update home-assistant app to 2021.12.0 (#1160)

lukechilds committed:
Update lnbits app to ec874ba (#1161)

yzernik committed:
Update squeaknode app to v0.1.182 (#1156)

lukechilds committed:
Update dashboard to v0.3.30

phaus committed:
Add uptime-kuma app (#1148)

louneskmt committed:
Add git to dependencies in readme (#1163)

yzernik committed:
Update squeaknode app repo and support urls (#1164)

aphex3k committed:
Add boltz service to ride-the-lightning app (#1123)



Co-authored-by: Luke Childs <lukechilds123@gmail.com>

louneskmt committed:
Add automatic decrypt and restore SCB script (#1162)

lukechilds committed:
Umbrel v0.4.10 (#1165)

mayankchhabra committed:
Update README with Umbrel 0.4.9

There's an issue with Umbrel 0.4.10 where new users don't see the seed phrase during setup.

yzernik committed:
Update squeaknode app to v0.1.186 (#1167)

daveajones committed:
Add helipad app (#1174)



Co-authored-by: Luke Childs <lukechilds123@gmail.com>

lukechilds committed:
Document `$APP_PASSWORD` and `$APP_SEED` env vars

1010Tom committed:
Upgrade itchysats to 0.3.2

lukechilds committed:
Add helipad registry.json entry

scratchscratchscratchy committed:
Upgrade to `0.3.3`

Includes using `$APP_PASSWORD` to restrict access to ItchySats within Umbrel by basic auth.

bonomat commented:
already merged https://github.com/getumbrel/umbrel/pull/1149
